mindsdb/integrations/handlers/tpot_handler/tpot_handler.py

```diff
@@ -33,8 +35,22 @@ def create(self, target: str, df: Optional[pd.DataFrame] = None, args: Optional[
         
 
         if df is not None:
+            # Separate out the categorical and non-categorical columns
+            categorical_cols=[col for col,type_col in type_of_cols.items() if type_col=='categorical']
```

Comment:
`from type_infer.dtype import dtype`

And then:

`if type_col in (dtype.categorical, dtype.binary)`

Reply:
this is not working